pepper-apis/src/appengine/Housekeeping.yaml

```diff
@@ -0,0 +1,14 @@
+service: housekeeping
+runtime: java11
+instance_class: B4
+# B2 allows background threads and we can use manual scaling
+manual_scaling:
+  instances: 1
+
+entrypoint: java -Xmx820m -Xmx820m -jar Housekeeping.jar
+
+# GAE does not like the "=" character in entrypoint and does not support "." in environment variables
+env_variables:
+  config_file: application.conf
+  ddp_firecloudKeysDir: fc_keys
+
```

Comment:
Note that we specify runtime config params here. GAE generates warning if you expand the command line to include params with the "-", e.g. `-DconfigFile=`

Reply:
Wait a minute.. is it only a warning and not an error?

Reply:
A warning to the effect of since you included these characters we will have to spawn a shell before we can launch your java process which slows down start up time (I forget the specific wording but got fed up with seeing that warning each time).